internal/operators/lvm/manifest.go

```diff
@@ -32,11 +35,33 @@ func Manifests() (map[string][]byte, []byte, error) {
 	return openshiftManifests, lvmcluster, nil
 }
 
-func getSubscription() ([]byte, error) {
+func getSubscription(cluster *common.Cluster) ([]byte, error) {
+
+	ocpVersion, err := version.NewVersion(cluster.OpenshiftVersion)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	minOCPVersionForLVM, err := version.NewVersion(lvmsMinOpenshiftVersion)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	if ocpVersion.LessThan(minOCPVersionForLVM) {
+		data := map[string]string{
+			"OPERATOR_NAMESPACE":              Operator.Namespace,
+			"OPERATOR_SUBSCRIPTION_NAME":      "odf-lvm-operator",
+			"OPERATOR_SUBSCRIPTION_SPEC_NAME": "odf-lvm-operator",
+		}
+		return executeTemplate(data, "LvmSubscription", LvmSubscription)
+	}
+
 	data := map[string]string{
-		"OPERATOR_NAMESPACE":         Operator.Namespace,
-		"OPERATOR_SUBSCRIPTION_NAME": Operator.SubscriptionName,
+		"OPERATOR_NAMESPACE":              Operator.Namespace,
+		"OPERATOR_SUBSCRIPTION_NAME":      "lvms-operator",
```

Comment:
please use const instead of "lvms-operator"

Reply:
declare const in one place, config file